examples/Bunch/include/simulation_defines/param/fieldBackground.param

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright 2014-2015 Axel Huebl, Alexander Debus
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/** Load pre-defined templates */
+#include "fields/background/templates/bgrTWTS.def"
+
+#ifndef PARAM_INCLUDE_FIELDBACKGROUND
```

Comment:
@BeyondEspresso you have become a very gentle developer in a steep curve, I like that! Nice and clean. :+1: